wondrous-app/components/Common/TaskSubmission/styles.tsx

```diff
@@ -105,52 +107,76 @@ export const SubmissionButtonEdit = styled(ButtonBase)`
   }
 `;
 
-export const SubmissionButtonRequestChange = styled(ButtonBase)`
-  && {
-    border: 1px solid #474747;
-    width: max-content;
+const ButtonStyleSelected = css`
+  ${({ selected }) =>
+    selected &&
+    `
+  > button {
+    background: ${palette.background.default};
+    :hover {
+      background: ${palette.background.default};
+    }
+  }
+ `}
+`;
+
+const ButtonStyleBase = css`
+  background: ${palette.grey78};
+  border-radius: 100px;
+  color: ${palette.white};
+  font-family: ${typography.fontFamily};
+  font-weight: 500;
+  height: 30px;
+  min-height: fit-content;
+  width: fit-content;
+  z-index: 1;
+  > button {
+    background: ${palette.grey900};
     height: 28px;
-    border-radius: 100px;
-    font-family: 'Space Grotesk';
-    padding: 7px 16px;
-    ${({ theme }) => `
-    color: ${theme.palette.white};
-    font-weight: ${theme.typography.fontWeightMedium};
-  `}
+    font-size: 14px;
+    line-height: 0;
+    font-family: ${typography.fontFamily};
+    font-weight: 500;
+    padding: 0 16px;
+    :hover {
+      background: ${palette.grey900};
+    }
   }
 `;
 
-export const SubmissionButtonReject = styled(ButtonBase)`
+export const SubmissionButtonRequestChange = styled(Button)`
   && {
-    border: 1px solid #474747;
-    width: max-content;
-    height: 28px;
-    border-radius: 100px;
-    border-color: ${palette.red400};
-    font-family: 'Space Grotesk';
-    padding: 7px 16px;
-    ${({ theme }) => `
-    color: ${theme.palette.white};
-    font-weight: ${theme.typography.fontWeightMedium};
-  `}
+    ${ButtonStyleBase}
+    ${ButtonStyleSelected}
+    background: ${({ selected }) =>
+      selected && `linear-gradient(270deg, ${palette.yellow800} -5.62%, ${palette.highlightPurple} 103.12%)`};
   }
 `;
 
-export const SubmissionButtonApprove = styled(ButtonBase)`
+export const SubmissionButtonReject = styled(Button)`
   && {
-    background: #474747;
-    width: max-content;
-    height: 28px;
-    border-radius: 100px;
-    font-family: 'Space Grotesk';
-    padding: 7px 16px;
-    ${({ theme }) => `
-    color: ${theme.palette.white};
-    font-weight: ${theme.typography.fontWeightMedium};
-  `}
+    ${ButtonStyleBase}
+    ${ButtonStyleSelected}
+    background: ${({ selected }) =>
+      selected && `linear-gradient(270deg, ${palette.red300} -5.62%, ${palette.highlightPurple} 103.12%)`};
+  }
+`;
+
+export const SubmissionButtonApprove = styled(Button)`
```

Comment:
Please use button from `components/Button`

Reply:
It already use `components/Button`